projects/packages/my-jetpack/_inc/components/connections-section/index.jsx

```diff
@@ -40,12 +38,15 @@ export default function ConnectionsSection() {
 		return getProductSlugsThatRequireUserConnection( products );
 	}, [ products, isLoading, isError ] );
 
+	const onConnectUser = useCallback( () => {
+		window.location.href = getMyJetpackUrl( '&step=connect-user' );
```

Comment:
`useMyJetpackNavigate` is using `useNavigate` hook from react-router. Since the project uses that, isn;t it better to use the hook rather than changing the location directly?

Reply:
Also, `useNavigate` works only for the internal routes, not the external redirects.